easybuild/easyconfigs/m/MMseqs2/MMseqs2-10-6d92c-intel-2019b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'MMseqs2'
+version = '10-6d92c'
+
+homepage = 'https://mmseqs.com'
+description = "MMseqs2: ultra fast and sensitive search and clustering suite"
+
+toolchain = {'name': 'intel', 'version': '2019b'}
```

Comment:
@Darkless012 Toolchain here can be lowered to `iimpi` (which is equivalent to `gompi` used in the existing `MMseqs2-10-6d92c-gompi-2019b.eb`)